frontends/dashboard/src/components/DatasetOverview.tsx

```diff
@@ -48,6 +50,57 @@ export const DatasetOverview = () => {
       setPage,
     });
 
+  const refetchChunks = async (datasetId: string) => {
+    try {
+      const api_host = import.meta.env.VITE_API_HOST as unknown as string;
+      const currentUsage = usage();
+      const prevCount = currentUsage[datasetId]?.chunk_count || 0;
+
+      const response = await fetch(`${api_host}/dataset/usage/${datasetId}`, {
+        method: "GET",
+        headers: {
+          "TR-Dataset": datasetId,
+          "Content-Type": "application/json",
+        },
+        credentials: "include",
+      });
+
+      if (!response.ok) {
+        throw new Error("Failed to fetch dataset usage");
+      }
+
+      const newData = await response.json();
```

Comment:
```suggestion
      const newData = await response.json() as {insert correct type from SDK types here}
```

Lint errors are coming because this type is unkown.